src/ResponseFetcher/LocalFrontendDrupalResponseFetcher.php

```diff
@@ -44,10 +44,11 @@ public function __construct($autoloader, $publicDir) {
    */
   public function fetchResponses(RequestInterface $frontend_request, RequestInterface $backend_request) {
     // Try finding frontend resource via filesystem.
-    $request_body = file_get_contents($this->publicDir . '/layout--default.html');
+    $path = $frontend_request->getUri()->getPath();
```

Comment:
right - we need to check for a frontend-path here and use it if it matches, else go with the default.

Reply:
that'S already handled by the request generator, so logic should be good here.